libc/src/gpu/rpc_open_port.cpp

```diff
@@ -0,0 +1,26 @@
+//===---------- GPU implementation of the external RPC port interface -----===//
```

Comment:
tiny nit: Your cpp files have the title too far to the right in the first line.

Reply:
Thanks for pointing that out.